design-at-cornell/src/design-community/design-faculty/Dashboard.tsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 import { HorizontalFlex, VerticalFlex } from '../../components/ContainerStyles';
-import { Grid } from '../../components/DashboardGridStyles';
+import { FacultyGrid } from '../../components/DashboardGridStyles';
 import { HeadingLine } from '../../components/DashboardElementStyles';
 import { Faculty } from '../../../../server/src/types';
 import FacultyBubble from './FacultyBubble';
 
-const Dashboard = (faculty: Faculty[]) => {
-  const facultyBubbles = (
-    <Grid>
+const FacultyImages = ({ faculty }: { faculty: Faculty[] }) => {
+  return (
+    <FacultyGrid>
       {Object.values(faculty)
         .slice(0, 6)
         .map((faculty) => (
-          <FacultyBubble key={faculty.name} {...faculty} />
+          <FacultyBubble faculty={faculty} key={faculty.name} />
```

Comment:
Removes prop spreading for this case-- if prop spreading is preferred that is fine but in general it seems clearer to pass props explicitly